### PR TITLE
docs: dagster_gcp.gcs.sensor.get_gcs_keys code example

### PR DESCRIPTION
**Pull request by mlarose**
## Summary & Motivation

Add documentation to get_gcs_keys.

## How I Tested These Changes

code example derived from what was used to test #20191

### CODE REVIEW

File: python_modules/libraries/dagster-gcp/dagster_gcp/gcs/sensor.py [hunk 1]
```diff
@@ -5,7 +5,52 @@
 
 
 def get_gcs_keys(bucket, prefix="", since_key=None, gcs_session=None):
-    """Credits to @ajohnson5 for the original implementation of this method."""
+    """Return a list of updated keys in a GCS bucket.
+
+    Credits to @ajohnson5 for the original implementation of this method.
```
Comment (erinkcochran87) [inline python_modules/libraries/dagster-gcp/dagster_gcp/gcs/sensor.py:10]: This feels odd to have in the docs. Can we add it as a code comment or in the PR instead?

**Review by erinkcochran87 — COMMENTED** [review]
Thanks for adding this! I assume these are supposed to show up in the API docs?

**Comment by mlarose** [comment]
> Thanks for adding this! I assume these are supposed to show up in the API docs?

Yes, it should show up similar to this: https://docs.dagster.io/_apidocs/libraries/dagster-aws#s3

File: python_modules/libraries/dagster-gcp/dagster_gcp/gcs/sensor.py (hunk 1 above)
Comment (mlarose) [reply python_modules/libraries/dagster-gcp/dagster_gcp/gcs/sensor.py:10]: I can, i did credit the author in the PR by linking to his and his name will also show up as co-author of the first commit.

**Comment by mlarose** [comment]
Screenshot 2024-03-13 at 9 51 09 AM

Finally got sphinx to work.

**Review by erinkcochran87 — APPROVED** [review]
Approving to unblock ya, but I noticed something small in the rendered API docs.

In the screenshot below, the `prefix` has only one double quote (`prefix="`), but in the code itself it looks right. I'm not sure if this is a docstring issue or a rendering issue, but if ya don't mind taking another look at it I'd appreciate it!

Screenshot 2024-03-13 at 1 01 04 PM

**Comment by mlarose** [comment]
> In the screenshot below, the prefix has only one double quote (prefix="), but in the code itself it looks right. I'm not sure if this is a docstring issue or a rendering issue, but if ya don't mind taking another look at it I'd appreciate it!


weird. sphinx or autodoc converts the double quotes to single quotes. it's valid code, but misleading, since you saw it as a double quote. I changed the default string param to None and adjusted the code and unit tests to match.

**Pull request merged**